simapp/v2/simdv2/cmd/config.go

```diff
@@ -104,3 +106,31 @@ func initCometOptions[T transaction.Tx]() cometbft.ServerOptions[T] {
 
 	return serverOptions
 }
+
+// appBuildingCommands are the commands which need a full SimApp to be built
+var appBuildingCommands = [][]string{
+	{"start"},
+	{"genesis", "export"},
+}
+
+// isAppRequired returns true if the command requires a full app to be built
```

Comment:
As this becomes a sort of necessary check, can we add this helper into v2?

Reply:
For my understanding you don't need the annotation anymore right? If so, let's just hardcode start in the helper, as it will always need an app.

As export is something you wire yourself, it makes sense to still require to provide it manually